ejson/src/client.ts

```diff
@@ -0,0 +1,39 @@
+import { Connection, WorkflowClient } from '@temporalio/client';
+import { v4 as uuid } from 'uuid';
+import type { Result, User } from './types';
+import { example } from './workflows';
+
+async function run() {
+  // @@@SNIPSTART typescript-ejson-client-setup
+  const client = new WorkflowClient(new Connection().service, {
+    dataConverter: { payloadConverterPath: require.resolve('./payload-converter') },
+  });
+  // @@@SNIPEND
+
+  // @@@SNIPSTART typescript-ejson-client
+  const user: User = {
+    id: uuid(),
+    // age: 1000n, BigInt isn't supported
+    hp: Infinity,
+    matcher: /.*Stormblessed/,
```

Comment:
Have you read them all??

Reply:
Everything in the Cosmere but the graphic novels!

Reply:
I only read the first 3 in the Stormlight Archive

Reply:
Number 4 is out, and I also really liked 2.5 and 3.5 (Edgedancer and Dawnshard)